frontend/src/component/ContentListItem.styl

```diff
@@ -9,6 +9,9 @@
   color fontColor
   height 60px
```

Comment:
Shouldn't the `min-height` be 60px? Then `height` can probably be removed.

Reply:
Indeed height could be set to 65px, removing the min-height property